pip/flatpak-pip-generator

```diff
@@ -56,16 +68,29 @@ def get_package_name(filename: str) -> str:
         segments = filename.split('-')
         if len(segments) == 5:
             return segments[0]
-        return '-'.join(segments[:len(segments) - 4])
+        candidate = segments[:len(segments) - 4]
+        # Some packages list the version number twice
```

Comment:
Just for ease of understanding maybe put an example.

Reply:
Added example for that line.